frontend/src/pages/projects/dataConnections/const.ts

```diff
@@ -41,6 +45,29 @@ export const AWS_FIELDS: FieldOptions[] = [
     label: 'Bucket',
   },
 ];
+export const PIPELINE_AWS_FIELDS: FieldOptions[] = [
```

Comment:
Can't we reuse `AWS_FIELDS`? This is a subset of `AWS_FIELDS` though.

Reply:
AWS_FIELDS have the variables sightly change , here all the fields are required